src/components/Board/Board.container.js

```diff
@@ -360,8 +361,19 @@ export class BoardContainer extends Component {
   };
 
   handleEditTileEditorSubmit = tiles => {
-    const { board, editTiles } = this.props;
+    const { intl, board, editTiles, showNotification } = this.props;
+
     editTiles(tiles, board.id);
+    showNotification(intl.formatMessage(messages.tilesEdited), [
+      <Button
```

Comment:
I would refactor this `Button` component to its own `UndoButton` in our shared UI components folder. This seems generic enough and many components will want to use this and it's nice to isolate the translation there as well.